packages/ui/src/components/Popover/index.tsx

```diff
@@ -0,0 +1,158 @@
+import { Tooltip } from '../Tooltip'
+
+type VariantType = 'default' | 'primary'
+
+const SIZES_WIDTH = {
+  small: 320,
+  medium: 420,
+  large: 520,
+}
+
+const StyledTooltip = styled(Tooltip, {
+  shouldForwardProp: prop => !['variant', 'size'].includes(prop),
+})<{
+  variant: VariantType
+  size: keyof typeof SIZES_WIDTH
+}>`
+  padding: ${({ theme }) => theme.space['2']};
+  width: ${({ size }) => SIZES_WIDTH[size]}px;
+  max-width: ${({ size }) => SIZES_WIDTH[size]}px;
+  text-align: initial;
+
+  ${({ theme, variant }) => {
+    if (variant === 'default') {
+      return `
+      background: ${theme.colors.neutral.background};
+      box-shadow: ${theme.shadows.popover};
+      &::after {
+        border-color: ${theme.colors.neutral.background} transparent transparent transparent;
+      }
+      `
+    }
+
+    return `
+      background: ${theme.colors.primary.backgroundStrong};
+      box-shadow: ${theme.shadows.popover};
+      &::after {
+        border-color: ${theme.colors.primary.backgroundStrong} transparent transparent transparent;
+      }
+      `
+  }}
+`
+
+const CloseButton = styled(Button)`
+  width: auto;
+  height: 24px;
+
+  &[data-variant='primary'] {
+    color: ${({ theme }) => theme.colors.neutral.iconStronger};
+  }
+`
+
+type ContentWrapperProps = Pick<
+  PopoverProps,
+  'title' | 'onClose' | 'variant' | 'children'
+>
+
+const ContentWrapper = ({
+  title,
+  onClose,
+  children,
+  variant,
+}: ContentWrapperProps) => (
+  <Stack gap={1}>
+    <Stack direction="row" justifyContent="space-between">
+      <Text
+        variant="headingSmall"
+        as="h3"
+        prominence={variant === 'default' ? 'strong' : 'stronger'}
+      >
+        {title}
+      </Text>
+      <CloseButton
+        onClick={onClose}
+        title="close"
+        variant="transparent"
+        icon="close"
+        iconSize={24}
+        action
+        data-variant={variant}
+      />
+    </Stack>
+    {typeof children === 'string' ? (
+      <Text
+        variant="bodySmall"
+        as="p"
+        prominence={variant === 'default' ? 'strong' : 'stronger'}
+      >
+        {children}
+      </Text>
+    ) : (
+      children
+    )}
+  </Stack>
+)
+
+type PopoverProps = {
+  children: ReactNode
+  content: ReactNode
+  title: string
+  variant?: VariantType
+  visible?: boolean
+  size?: keyof typeof SIZES_WIDTH
+  onClose?: () => void
+  className?: string
+  'data-testid'?: string
+} & Pick<ComponentProps<typeof Tooltip>, 'placement'>
+
+export const Popover = ({
+  visible = false,
+  children,
+  placement,
+  content,
+  title,
+  variant = 'default',
+  size = 'medium',
+  onClose,
+  className,
+  'data-testid': dataTestId,
+}: PopoverProps) => {
+  const [opened, setOpened] = useState(visible)
+
+  useEffect(() => {
+    setOpened(visible)
+  }, [visible])
+
+  const onCloseCallBack = useCallback(() => {
+    setOpened(false)
+    onClose?.()
+  }, [onClose])
+
+  return (
+    <StyledTooltip
```

Comment:
it's not really a tooltip anymore so the role need to be dialog and not tooltip ?

Reply:
https://github.com/scaleway/scaleway-ui/pull/2378/files#diff-4b090bb740787ad99908e13624821cbfbd710c1700a550b5a00603a6e05057c3L297

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role